app/models/manageiq/providers/google/cloud_manager.rb

```diff
@@ -0,0 +1,52 @@
+class ManageIQ::Providers::Google::CloudManager < ManageIQ::Providers::CloudManager
+  require_nested :Refresher
+
+  def self.ems_type
+    @ems_type ||= "gce".freeze
+  end
+
+  def self.description
+    @description ||= "Google Compute Engine".freeze
+  end
+
+  def self.hostname_required?
+    false
+  end
+
+  def self.region_required?
+    false
+  end
+
+  def supported_auth_types
+    %w(oauth)
+  end
+
+  def supports_authentication?(authtype)
+    supported_auth_types.include?(authtype.to_s)
+  end
+
+  validates :provider_region, :inclusion => {:in => ManageIQ::Providers::Google::Regions.names}
+
+  def description
+    ManageIQ::Providers::Google::Regions.find_by_name(provider_region)[:description]
+  end
+
+  def verify_credentials(_auth_type = nil, _options = {})
+    true
+  end
+
+  def valid?(_options = {})
+    true
```

Comment:
I think I found the cause of the failing test

> ```
> 1) ExtManagementSystem validates across sub-classes duplicate name
>      Failure/Error: expect { FactoryGirl.create(t, :name => @ems.name, :hostname => @different_host_name, :zone => @zone) }.to raise_error(ActiveRecord::RecordInvalid, "Validation failed: Name has already been taken")
>        expected ActiveRecord::RecordInvalid with "Validation failed: Name has already been taken" but nothing was raised
>      # ./spec/models/ext_management_system_spec.rb:157:in `block (5 levels) in <top (required)>'
>      # ./spec/models/ext_management_system_spec.rb:156:in `each'
>      # ./spec/models/ext_management_system_spec.rb:156:in `block (4 levels) in <top (required)>'
> 
> ```

This is going to force all validations to be skipped.  I removed this method in my local copy of this PR and it got me half way there.

I also had to update the [factory for `:ems_google`](https://github.com/ManageIQ/manageiq/pull/4521/files#r43016186) to set a valid `provider_region` value.

And that fixed the test for me locally.